spirit/category/models.py

```diff
@@ -27,6 +27,8 @@ class Category(models.Model):
 
     # topic_count = models.PositiveIntegerField(_("topic count"), default=0)
 
+    colour = models.CharField(_("colour"), max_length=7, blank=True)
```

Comment:
Can you change this to `null=True, blank=True`?
For consistency with the rest of charfields. I usually prefer saving null in the DB instead of a empty string, so I can later query for None (makes more sense to me for some reason).